README.rdoc

```diff
@@ -54,7 +52,7 @@ including SSL connection verification, header handling and tunable options.
 
 == INSTALL:
 
-  gem install net-http-persistent
+    gem install net-http-persistent
```

Comment:
I'm not sure what this change is for, GitHub is rendering this as a `<pre>` block already:

<img width="792" alt="net-http-persistent 2019-05-06 17-30-09" src="https://user-images.githubusercontent.com/9831/57263166-9d32da80-7024-11e9-9b3f-98f82516d176.png">

Reply:
just for consistency since the other block had 4 indents

Reply:
Ah, my bad, I didn't expand more context.  It's good